config/webpack.config.js

```diff
@@ -405,6 +407,7 @@ module.exports = async (env = {}) => {
       new webpack.DefinePlugin({
         __BUILDTYPE__: JSON.stringify(buildtype),
         __API__: JSON.stringify(buildOptions.api),
+        __REGISTRY__: JSON.stringify(appRegistry),
```

Comment:
Good approach, I like this method of making the registry available to the function without having to read and parse the file again